Formula/opencl-headers.rb

```diff
@@ -21,24 +21,38 @@ class OpenclHeaders < Formula
 
   depends_on "cmake" => :build
 
+  resource "opencl-clhpp-headers" do
+    url "https://github.com/KhronosGroup/OpenCL-CLHPP/archive/refs/tags/v2022.09.30.tar.gz"
+    sha256 "999dec3ebf451f0f1087e5e1b9a5af91434b4d0c496d47e912863ac85ad1e6b2"
+  end
```

Comment:
Isn't this just a different project?

Reply:
OpenCL C and C++ headers are sometimes distributes in a single (CentOS 7 as an example), sometimes as two packages where C++ depends on C.

Do you think it's better to create a dedicated package for C++ headers which depends on C?

Reply:
Seeing how upstream treats it as two projects I think Homebrew should too.

Reply:
done, thanks for the suggestion.